docs/source/dev-and-test/setup-run-node.md

```diff
@@ -38,3 +38,41 @@ The BigchainDB [CONTRIBUTING.md file](https://github.com/bigchaindb/bigchaindb/b
 Ian Worrall of [Encrypted Labs](http://www.encryptedlabs.com/) wrote a document (PDF) explaining how to set up a BigchainDB (Server) dev machine on [Cloud9](https://c9.io/):
 
 [Download that document from GitHub](https://github.com/bigchaindb/bigchaindb/raw/master/docs/source/_static/cloud9.pdf)
+
+
+## Option C: Using a Local Dev Machine and Docker
+
+You need to have recent versions of [docker engine](https://docs.docker.com/engine/installation/#installation)
+and [docker-compose](https://docs.docker.com/compose/install/), and possibly
+[docker-machine](https://docs.docker.com/machine/install-machine/),`if you are
```

Comment:
Yes, I know, but I wanted to avoid to link to platform specific pages, as docker supports other platforms as well. From the links above the user can figure out what to do next depending on which platform they are.  Does this make sense?